Patch 1

gavit opened the pull request:
Sanitized code to protect against potential security issues.

Commit:
Update vpn_openvpn_export.php

Windows installers reflect the target OS it's for.
e.g.: x86-win6, x86-xp, x64-win6, x64-xp

Commit:
Update vpn_openvpn_export.php

Sanitized the code as requested to prevent potential security issues

jim-p commented:
It's better, but not ideal. Lots of unnecessarily duplicated code. With some better structure that could be reduced significantly. Also there would be no need to have "inst-" and "install" in the filename. If you're hardcoding it, may as well reduce the redundancy.

rbgarga commented:
In order to have patches accepted you must sign CLA as described at https://www.pfsense.org/about-pfsense/#legal

gavit commented:
Please see attached,

Kind regards,
Gabor

On Thu, Sep 24, 2015 at 9:54 AM, Renato Botelho <notifications@github.com>
wrote:

> In order to have patches accepted you must sign CLA as described at
> https://www.pfsense.org/about-pfsense/#legal
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pfsense/pfsense-packages/pull/1052#issuecomment-142918920>
> .
>

Commit:
Update vpn_openvpn_export.php

Removed unnecessarily duplicated code. Also there would be removed "inst-" and "install" in the filename.

phil-davis commented on config/openvpn-client-export/vpn_openvpn_export.php:
```diff
@@ -252,8 +252,20 @@
-		$exp_name = urlencode($exp_name."-install.exe");
+	if (substr($act, 0, 6) == "inst-x") {
+		if (substr($act, 6, 9) == "86-") {
+			if (substr($act, 9, 11) == "xp) {
```
Missing closing double-quotes.
Same syntax error in 4 places.

rbgarga commented:
This repository is obsolete, it was used only for pfSense 2.2 and earlier versions.

For current pfSense versions, packages are hosted at https://github.com/pfsense/freebsd-ports
